pymilvus/client/grpc_handler.py

```diff
@@ -613,8 +613,9 @@ def _execute_search_requests(self, requests, timeout=None, **kwargs):
                     raise BaseException(response.status.error_code, response.status.reason)
 
                 raws.append(response)
-
-            return ChunkedQueryResult(raws, auto_id)
+            # tag edit
+            round_decimal = kwargs.get("round_decimal",3)
```

Comment:
If the user does not pass in the round_decimal parameter, it should be processed according to the original logic. Instead of using 3 as the default value.

Reply:
in milvus that i edited version ,when go send search parameter to C++，gprc need round_decimal this parameter. And Go also has code to handle round_decimal. So can i set default round_decimal -1 ? if milvus find round_decimal value is -1, milvus will do precision control.

Reply:
This is also a solution.